### ISSUE TITLE
OpenVSX marketplace?

### ISSUE THREAD

**Issue by jabirali** [aioutecism/amVim-for-VSCode#278]
First of all, thanks for making this extension! :heart: 

Some people use [VSCodium](https://github.com/VSCodium/), a pure open-source version of VSCode with telemetry disabled, instead of the Microsoft binaries. For license reasons, these non-official builds cannot use the official VSCode Marketplace; see [this](https://github.com/VSCodium/vscodium#more-info) and [this](https://github.com/VSCodium/vscodium/blob/master/DOCS.md#extensions--marketplace) for more details.

In practice, VSCodium users can install plugins from the marketplace [open-vsx.org](https://open-vsx.org/) straight from the app, but for plugins like amVim that are only hosted on the Microsoft marketplace, we have to manually download a `.vsix` file from the website and install that.

Would you kindly consider hosting this on open-vsx.org as well? (Currently, VSCodeVim is in there, but amVim is unfortunately not, and I prefer this plugin.) I am not familiar with the publishing process myself, but it seems to be documented [here](https://github.com/VSCodium/vscodium/blob/master/DOCS.md#extensions-marketplace) and [here](https://github.com/eclipse/openvsx/wiki/Publishing-Extensions).

**Comment by alisonatwork** [comment]
Hi @jabirali - I haven't used VSCodium before, but it seems like a reasonable project. Currently the release process for amVim is managed by @aioutecism. If we were to add a second publishing destination, they would be the right person to approve that.

**Comment by timescam** [comment]
Sorry for necroing this issue, but I also love to install amVim form OpenVSX

**Comment by alisonatwork** [comment]
Thanks for the reminder! I will take a look at this when I have a bit more time. I am currently long-term traveling and do not have regular internet access.

**Comment by 9j** [comment]
OpenVSX support is also required in [gitpod.io](https://gitpod.io/)

**Comment by 9j** [comment]
it seems there is no problem with the [ISC License](https://github.com/aioutecism/amVim-for-VSCode/blob/9ff8d28ea292ea63193259ba5b0c8453e75bf050/package.json#L32)
so I fork this project and [publish the extension to OpenVSX](https://open-vsx.org/extension/9j/amvim)

Please let me know if it is a problem.
Thank you.

**Comment by yashpalgoyal1304** [comment]
jjust pasting the original extension id for quick reference: [auiworks.amvim](https://marketplace.visualstudio.com/items/auiworks.amvim)

@9j thanks for doing the lifting. umh, but do u agree to remove it once the official one is online on open-vsx??

**Comment by 9j** [comment]
> 

Anytime, I'll stop posting if they officially support open-vsx.

**Comment by alisonatwork** [comment]
Hi everyone! Sorry for the long delay on this. I have finally published 1.34.0 (last version) to OpenVSX. I pushed it up there so that anybody using it can get set up with that version now ahead of our 1.35.0 release which will include more supported for navigating across wrapped lines (see #116 and #303). I really appreciate you getting set up with Eclipse.org and publishing the fork in the meanwhile, @9j, and again I apologize for not being faster on this.
